moveit_planners/pilz_industrial_motion_planner/include/pilz_industrial_motion_planner/trajectory_generator.h

```diff
@@ -271,7 +279,7 @@ class TrajectoryGenerator
 protected:
   const moveit::core::RobotModelConstPtr robot_model_;
   const pilz_industrial_motion_planner::LimitsContainer planner_limits_;
-  static constexpr double MIN_SCALING_FACTOR{ 0.0001 };
+  static constexpr double MIN_SCALING_FACTOR{ 0.0000001 };
```

Comment:
This may end up changing default behavior -- but as a question to you, did you find this made a difference in other cases?

Also minor nitpick, with all the zeros consider using scientific notation like `1e-7`.